jwt_auth/authentication.py

```
@@ -61,7 +61,7 @@ def _get_user(cls, token: str) -> User:
         Returns:
             The user who made the request
         """
-        keys = settings.JWT_AUTH['KEYS']
+        keys = settings.JWT_AUTH.get('KEYS', [])
```

so the reason for this was that it should fail early if the keys are not set. it will still fail but the error will be a signature error which is a bit misleading I think.